roles/ceph-client/defaults/main.yml

```diff
@@ -1,18 +0,0 @@
----
```

Comment:
@guits yeah, I think it might make sense to combine ``ceph-config`` and ``ceph-defaults``. I guess I just don't see the use case for ``ceph-defaults`` on it's own. Seems like you'd always want to use ``ceph-config`` so that ``fsid`` is populated correctly.

I do like the behavior added with ``no_deps``. I still think that the configuration for all roles besides ``ceph-common`` and ``ceph-docker-common`` should stay with the respective role though. Anything that is common across roles or needed for ceph.conf generation should be included in ``ceph-config`` or ``ceph-defaults``.